plugins/pose_prediction/plugin.cpp

```diff
@@ -201,104 +197,6 @@ class pose_prediction_impl : public pose_prediction {
     mutable Eigen::Quaternionf                                       offset{Eigen::Quaternionf::Identity()};
     mutable std::shared_mutex                                        offset_mutex;
 
-    // Slightly modified copy of OpenVINS method found in propagator.cpp
```

Comment:
The code LGTM. But why do we want to move the RK4 code from the `rk4_integrator` plugin, to our include folder instead?